Documentation/ExtensionArchitecture/FileStructure/Resources/Private/Index.rst

```diff
@@ -3,27 +3,149 @@
+
+    Folder `Layouts` often contains the Fluid layouts for a TYPO3 extension.
+    These can be included via the `Layout ViewHelper <f:layout> <https://docs.typo3.org/permalink/t3viewhelper:typo3fluid-fluid-layout>`_
+    into the main Fluid template.
+
+.. _extension-resources-private-fluid-site-packages:
+
+Common Fluid template locations for site packages
+=================================================
+
+Commonly site package in TYPO3 v13 and above use the `PAGEVIEW <https://docs.typo3.org/permalink/t3tsref:cobj-pageview>`_
+TypoScript object to display the HTMl page output. They have one folder, commonly
+`PageView` or `Templates` in folder `Resources/Private` with the subfolders
+`Pages`, `Partials` and `Layouts` (they cannot be renamed).
+
+..  typo3:file:: MyPageLayout.html
+    :scope: extension
+    :path: /Resources/Private/PageView/Pages$Z/
+    :regex: /^.*\/Resources\/Private\/PageView\/Pages\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Templates for different page layouts
+
+    This folder contains one Fluid template for each page layout defined in the
+    site package. See
+
+..  typo3:file:: SomePartials.html
+    :scope: extension
+    :path: /Resources/Private/PageView/Partials/
+    :regex: /^.*\/Resources\/Private\/PageView\/Partials\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Partials for the page view
+
+    Folder `Partials` often contains the Fluid partials for a TYPO3 extension.
+    these can be included via the `Render ViewHelper <f:render> <https://docs.typo3.org/permalink/t3viewhelper:typo3-fluid-render>`_
+    into the main Fluid template.
+
+..  typo3:file:: SomeLayout.html
+    :scope: extension
+    :path: /Resources/Private/PageView/Layouts/
+    :regex: /^.*\/Resources\/Private\/PageView\/Layouts\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Layouts for the page view
+
+    Folder `Layouts` often contains the Fluid layouts for a TYPO3 extension.
+    these can be included via the `Layout ViewHelper <f:layout> <https://docs.typo3.org/permalink/t3viewhelper:typo3fluid-fluid-layout>`_
+    into the main Fluid template.
+
+Templates to override or extend Fluid-Styled Content based content objects are
+typically stored in a folder called `/Resources/Private/ContentElements`:
+
+..  typo3:file:: MyPageLayout.html
+    :scope: extension
+    :path: /Resources/Private/ContentElements/Pages/
+    :regex: /^.*\/Resources\/Private\/ContentElements\/Pages\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Templates for different content elements
+
+    This folder contains one Fluid template for each page layout defined in the
+    site package. See
+
+..  typo3:file:: SomePartials.html
+    :scope: extension
+    :path: /Resources/Private/ContentElements/Partials/
+    :regex: /^.*\/Resources\/Private\/ContentElements\/Partials\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Partials for content elements
+
+    Folder `Partials` often contains the Fluid partials for a TYPO3 extension.
```

Comment:
Same as above. If you want to keep it, maybe a note that also partials of PAGEVIEW and/or Extbase Partials can be used here.

Reply:
You wouldnt put PAGEVIEW partials in path ContentElements/Partials, would you?